cli/main.go

```diff
@@ -197,13 +186,17 @@ func main() {
 						proof, err := core.LoadCheckpointProofFromFile(checkpointProofPath)
 						core.PanicOnError("failed to parse checkpoint proof from file", err)
 
-						submitCheckpointProof(*owner, eigenpodAddress, chainId, proof, eth)
+						core.SubmitCheckpointProof(ctx, *owner, eigenpodAddress, chainId, proof, eth, batchSize, noPrompt)
 						return nil
 					}
 
 					currentCheckpoint := core.GetCurrentCheckpoint(eigenpodAddress, eth)
 					if currentCheckpoint == 0 {
 						if owner != nil {
+							if !noPrompt {
+								core.PanicIfNoConsent("This will start a checkpoint on your eigenpod.")
+							}
```

Comment:
I just tried this - and I don't think it's quite sufficient. Since you can't cancel a checkpoint, starting a checkpoint also requires you to complete the checkpoint.

So this current code asks me for consent twice:
![image](https://github.com/user-attachments/assets/7abe1777-4858-45e1-948b-bfca01a63991)

Reply:
IMO that first consent question is kinda useless because it doesn't tell me how many txns I will need to do to complete the checkpoint, or how many shares I'll get by completing it, or what the gas required to complete it will be.

Reply:
I would remove the second consent question and move it all into the first question

Reply:
Updated the logic here